tests/fsharp/new.fs

```diff
@@ -0,0 +1,171 @@
+module M
+
+type ClassPublic() = class end
+
+type private ClassPrivate() =   // CUT
+   static member ClassPrivateProperty = 1   // CUT
+
+type InterfacePublic = interface end
+
+type private InterfacePrivate = interface end
```

Comment:
@dsyme is my understanding correct here that this should be kept only when there is an `InternalsVisibleTo` present on the assembly level?